src/middlewared/middlewared/plugins/cloud_backup/crud.py

```diff
@@ -149,6 +151,10 @@ async def _validate(self, app, verrors, name, data):
         if data["snapshot"] and data["absolute_paths"]:
             verrors.add(f"{name}.snapshot", "This option can't be used when absolute paths are enabled")
 
+        if data["cache_path"]:
+            await check_path_resides_within_volume(verrors, self.middleware, f"{name}.cache_path", data["cache_path"],
```

Comment:
Should we check whether path is readonly? What about if path is case-insensitive but source is case-sensitive?

Reply:
> Should we check whether path is readonly?

How?

> What about if path is case-insensitive but source is case-sensitive?

Why would it matter?

Reply:
> > Should we check whether path is readonly?
> 
> How?

```
root@testIKNJTCYLN8[~/middleware/src/middlewared]# midclt call filesystem.statfs /mnt/dozer/foo | jq
{
  "flags": [
    "RO",
    "NOATIME",
    "XATTR",
    "POSIXACL",
    "CASESENSITIVE"
  ],
  ```
  `RO` in `flags`?
  
  > Why would it matter?
  Not sure. I'm unfamiliar with restic internals regarding whether case-sensitivity of cache path matters.

Reply:
> I'm unfamiliar with restic internals regarding whether case-sensitivity of cache path matters.

It's cache structure seems to be just hexadecimal strings, so it should be fine